requirements.txt

```diff
@@ -0,0 +1,11 @@
+-e .
+colour-runner==0.0.4
+coverage==4.2
+dj-database-url==0.4.1
+django>=1.8,<1.9
+factory-boy==2.7.0
+flake8-import-order==0.9.2
+flake8==3.0.4
+incuna-test-utils==6.4.1
+mock==2.0.0
```

Comment:
Oh I see, ignore my last comment, it seems that it's required by `incuna-test-utils`. Maybe a note should be added in the `readme` of `incuna-test-utils` about it.